app/src/main/java/cm/aptoide/pt/home/FakeBundleDataSource.java

```diff
@@ -13,19 +13,29 @@
 
 public class FakeBundleDataSource implements BundleDataSource {
 
-  @Override public Single<HomeBundlesModel> loadFreshHomeBundles() {
+  @Override public Single<HomeBundlesModel> loadFreshHomeBundles(String key) {
     return getHomeBundles();
   }
 
-  @Override public Single<HomeBundlesModel> loadNextHomeBundles(int offset, int limit) {
-    return loadFreshHomeBundles();
+  @Override public Single<HomeBundlesModel> loadNextHomeBundles(int offset, int limit, String key) {
+    return loadFreshHomeBundles(key);
   }
 
-  @Override public boolean hasMore(Integer offset) {
+  @Override public boolean hasMore(Integer offset, String title) {
     return true;
   }
 
-  public Single<HomeBundlesModel> getHomeBundles() {
+  @Override public Single<HomeBundlesModel> loadFreshBundleForEvent(String url, String key) {
+    return null;
```

Comment:
This should return the getHomeBundles model, can be useful for testing for example.